docs/smart-contracts/getting-started/create-an-app.mdx

```diff
@@ -425,6 +425,22 @@ If you're still having problems, join our Discord (link above) or [open an issue
 
 This means that Testnet is down, and you probably just need to wait a while and try again.
 
+### Freigther popup mentioning SSL required
+
+- Install a vite plugin: `npm install --save @vitejs/plugin-basic-ssl`
+- Edit `astro.config.mjs`:
+
+```mjs
```

Comment:
I don't think `mjs` is a valid prism language. Let's change this to `js` or `javascript`